day 2/ans1.js

```diff
@@ -0,0 +1,22 @@
+//Write a function that can stop execution of a function for the number of milliseconds
+// sent as an argument
+// Example:
+// const func = async () => {
+// console.log("Printing before")
+// //Call your function here eg. sleep(3000)
+// console.log("Printing after")
+// }
+
+
+const ansFn=new Promise((resolve)=>{
+    setTimeout(resolve,3000)
+})
+
+const func = async () => {
+console.log("Printing before")
+await ansFn.then()
+console.log("Printing after")
+}
+
+
+func()
```

Comment:
You don't need to chain .then() inside an await statement because await will handle that behavior for you.